prisma/schema.prisma

```diff
@@ -7,7 +7,7 @@ generator client {
 
 datasource db {
   provider = "postgresql"
-  url      = env("DATABASE_URL")
+  url = env("TEST_DATABASE_URL")
```

Comment:
Is this for updating the test database? We should probably have a switch in here that checks the local environment and changes the env based on that, rather than manually changing the url?